ext/standard/tests/bug79410.phpt

```diff
@@ -0,0 +1,10 @@
+--TEST--
+Bug #79410 (system() swallows last chunk if it is exactly 4095 bytes without newline)
+--FILE--
+<?php
+    ob_start();
+    system(getenv('TEST_PHP_EXECUTABLE') . " -n -r 'echo str_repeat(\".\", 4095);'");
```

Comment:
This doesn't work on Windows, because only double-quotes are recognized as argument enclosures. I think in this case swapping the quotes may work on all supported platforms:
```suggestion
    system(getenv('TEST_PHP_EXECUTABLE') . ' -n -r "echo str_repeat(\".\", 4095);"');
```